src/main/java/uk/gov/hmcts/reform/notificationservice/service/NotificationMessageProcessor.java

```diff
@@ -38,7 +38,13 @@ public NotificationMessageProcessor(
      * @return false if there was no message to process. Otherwise true.
      */
     public boolean processNextMessage() throws ServiceBusException, InterruptedException {
+        log.info("Polling the queue");
         IMessage message = messageReceiver.receive();
+        log.info("Polled the queue, message is null: {}, id: {}",
+            message == null,
+            ((message == null) ? "" : message.getMessageId())
+        );
+
```

Comment:
do we need this? we log this stuff below

Reply:
from microsoft guy : `Message got picked up and locked at  6/23/2020 ​12:24:52 PM: 

Id   = 30461
ProcessId  = 25104
TimeCreated  = 6/23/2020 ​12:24:52 PM
Task   = MessageCursorBufferLockMessage
Message  = TrackingId: 8153bd8a0000001f001429135eec720b_G35_B14, lock message: entityName: reform-scan-servicebus-prod:Queue:notifications, SubQueue: 0,  sequenceNumber: 104, messageId: 76c7ab85-2024-4d12-91c2-55244e53c1c4, sessionId: null, partitionId: -1, lockToken: 89981af0-eac2-4d6f-83c7-678e656f0236
`

and this what we log below at `INFORMATION1:33:45.979 PM`
Started processing notification message with ID 76c7ab85-2024-4d12-91c2-55244e53c1c4

Reply:
we still have following logs which can serve same purpose as this log entry ;)

Reply:
if the message null?
trying to understand how long does it take a get a message from queue

Reply:
if message is null - there's another log

Reply:
other one is debug